wagtail/documents/views/bulk_actions/document_bulk_action.py

```diff
@@ -0,0 +1,19 @@
+from wagtail.admin.views.bulk_action import BulkAction
+from wagtail.documents import get_document_model
+from wagtail.documents.permissions import permission_policy as documents_permission_policy
+
+
+class DocumentBulkAction(BulkAction):
+    permission_policy = documents_permission_policy
+    model = get_document_model()
+    object_key = 'document'
+
+    def get_context_data(self, **kwargs):
+        ctx = super().get_context_data(**kwargs)
```

Comment:
```suggestion
        context = super().get_context_data(**kwargs)
```

`ctx` -> `context`
a tad more vebose, but easier to parse